arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,1041 @@
+  },
+  "definitions": {
+    "CertificateDescription": {
+      "description": "Certificate description",
+      "properties": {
+        "thumbprint": {
+          "type": "string"
+        },
+        "thumbprintSecondary": {
+          "type": "string"
+        },
+        "x509StoreName": {
+          "type": "string",
+          "enum": [
+            "AddressBook",
+            "AuthRoot",
+            "CertificateAuthority",
+            "Disallowed",
+            "My",
+            "Root",
+            "TrustedPeople",
+            "TrustedPublisher"
+          ]
+        }
+      }
+    },
+    "SettingsParameterDescription": {
+      "description": "Settings parameter description",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of settings parameter description"
+        },
+        "value": {
+          "type": "string",
+          "description": "The value of the property"
+        }
+      }
+    },
+    "SettingsSectionDescription": {
+      "description": "Settings section description",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of settings section description"
+        },
+        "parameters": {
+          "description": "Settings parameter description",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SettingsParameterDescription"
+          }
+        }
+      }
+    },
+    "EndpointRangeDescription": {
+      "description": "Endpoint range description",
+      "properties": {
+        "startPort": {
+          "type": "integer"
+        },
+        "endPort": {
+          "type": "integer"
+        }
+      }
+    },
+    "NodeTypeDescription": {
+      "description": "NodeType description",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "placementProperties": {
+          "description": "Placement properties key",
+          "type": "object",
+          "additionalProperties": {
+            "description": "Placement properties value",
+            "type": "string"
+          }
+        },
+        "capacities": {
+          "description": "Capacities key",
+          "type": "object",
+          "additionalProperties": {
+            "description": "Capacities value",
+            "type": "string"
+          }
+        },
+        "clientConnectionEndpointPort": {
+          "description": "Client connection endpoint port",
+          "type": "integer"
+        },
+        "httpGatewayEndpointPort": {
+          "description": "Http gateway endpoint port",
+          "type": "integer"
+        },
+        "durabilityLevel": {
+          "description": "Nodetype durability Level",
+          "type": "string",
+          "enum": [
+            "Bronze",
+            "Silver",
+            "Gold",
+            "Platimun"
+          ]
+        },
+        "applicationPorts": {
+          "description": "Application ports",
```

Comment:
If this is the range of public ports used by the deployed applications to listen for incoming traffic from the Internet, then please add that info to description